configs/default/SJET-AOCODAF722MINIV2.config

```diff
@@ -0,0 +1,162 @@
+# version
+# Betaflight / AOCODAF722MINIV2 (AO7M) 4.3.2 MAY 5 2023 / 10:01:04 () MSP API: 1.43
+
+
+board_name AOCODAF722MINIV2
+manufacturer_id SJET
+
+# resources
+resource BEEPER 1 C13
+resource MOTOR 1 B04
+resource MOTOR 2 B05
+resource MOTOR 3 B00
+resource MOTOR 4 B01
+resource MOTOR 5 C08
+resource MOTOR 6 C09
+resource MOTOR 7 B06
+resource MOTOR 8 B07
+resource PPM 1 A10
+resource LED_STRIP 1 A08
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_TX 3 B10
+resource SERIAL_TX 4 A00
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource SERIAL_RX 3 B11
+resource SERIAL_RX 4 A01
+resource SERIAL_RX 6 C07
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+resource LED 1 A13
+resource SPI_SCK 1 A05
+resource SPI_SCK 2 B13
+resource SPI_SCK 3 C10
+resource SPI_MISO 1 A06
+resource SPI_MISO 2 B14
+resource SPI_MISO 3 C11
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 2 B15
+resource SPI_MOSI 3 C12
+resource ADC_BATT 1 C02
+resource ADC_RSSI 1 C00
+resource ADC_CURR 1 C01
+resource FLASH_CS 1 D02
+resource OSD_CS 1 B12
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 B02
+resource USB_DETECT 1 C14
+
+# timer
+timer A10 AF1
+# pin A10: TIM1 CH3 (AF1)
+timer B04 AF2
+# pin B04: TIM3 CH1 (AF2)
+timer B05 AF2
+# pin B05: TIM3 CH2 (AF2)
+timer B00 AF2
+# pin B00: TIM3 CH3 (AF2)
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer C08 AF3
+# pin B03: TIM8 CH3 (AF1)
+timer C09 AF3
+# pin A15: TIM8 CH4 (AF1)
+timer B06 AF2
+# pin B06: TIM4 CH1 (AF2)
+timer B07 AF2
+# pin B07: TIM4 CH2 (AF2)
+timer A08 AF1
+# pin A08: TIM1 CH1 (AF1)
+
+# dma
+dma ADC 1 0
+# ADC 1: DMA2 Stream 0 Channel 0
+dma pin A10 0
+# pin A10: DMA2 Stream 6 Channel 0
+dma pin B04 0
+# pin B04: DMA1 Stream 4 Channel 5
+dma pin B05 0
+# pin B05: DMA1 Stream 5 Channel 5
+dma pin B00 0
+# pin B00: DMA1 Stream 7 Channel 5
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+
+dma pin C08 1
+# pin B03: DMA2 Stream 4 Channel 7
+dma pin C09 0
+# pin A15: DMA2 Stream 7 Channel 7
+dma pin B06 0
+# pin B06: DMA1 Stream 0 Channel 2
+dma pin B07 0
+# pin B07: DMA1 Stream 3 Channel 2
+dma pin A08 2
+# pin A08: DMA2 Stream 3 Channel 6
+
+# serial
+serial 20 1 115200 57600 0 115200
+serial 0 64 115200 57600 0 115200
+
+# feature
+feature RX_SERIAL
+feature TELEMETRY
+feature LED_STRIP
+feature OSD
+feature AIRMODE
+feature ANTI_GRAVITY
+feature GPS
+
+
+# master
+set motor_pwm_protocol = DSHOT600
+set serialrx_provider = SBUS
+set gyro_to_use = FIRST
+set align_mag = DEFAULT
+set mag_align_roll = 0
+set mag_align_pitch = 0
+set mag_align_yaw = 0

```

Comment:
Please don't open new PR's for the same work.

There are many defaults which needs to be removed.
The file has illegal DOS line endings.

Reply:
I have noticed this problem, thank you for reminding me

Reply:
@aocodarc , there are may ways to fix branches/PR'
please look into `git reset [--soft|--hard]` and/or rebasing and/or squashing.
once fixed you can force push (`git push --force`).


also this may be helpful for your overall git environment:
```
git config --global core.autocrlf true    ## use "input" for Linux or use "true" for Windows
git config --global core.safecrlf warn
git config --global core.whitespace cr-at-eol
git config --global core.filemode false
git config --global help.autocorrect true
```